app/utils/img_processing/google_api.py

```diff
@@ -43,7 +43,7 @@ def __init__(self):
             raise RuntimeError("Missing Google Credentials, Exiting app")
 
         # init the client for use by the functions
-        self.client = vision.ImageAnnotatorClient()
+        self.client = vision_v1.ImageAnnotatorClient()
 
         self.text_moderator = BadWordTextModerator(
             path.join(
```

Comment:
Does this fix the Google Vision API? This seems off topic to setting up a database connection. Maybe put this in a separate PR? (make a new commit that puts this file back the the way it was, create a new branch, paste these changes, commit them to the new branch, and make a PR for it.

Reply:
When I pulled from the main repo it wouldn't deploy locally and threw an error saying that this module could not be resolved. Upon research it appears Google has made some changes to the vision API and that the types should now be imported directly from google.cloud. I'll revert and make a separate PR for that. Thanks!